ui/app/AppLayouts/Profile/stores/WalletStore.qml

```diff
@@ -29,6 +33,21 @@ QtObject {
     property var overview: walletSectionOverview
     property var accounts: Global.appIsReady? accountsModule.accounts : null
     property var originModel: accountsModule.keyPairModel
+    property var ownAccounts: SortFilterProxyModel {
+        sourceModel: root.accounts
+        proxyRoles:  FastExpressionRole {
+            name: "preferredSharingChainShortNames"
+            expression: {
+                return root.networksModule.getNetworkShortNames(preferredSharingChainIds)
```

Comment:
imho it would be better if you returned the short names already from the model, instead of creating an `ExpressionRole` and calling back `getNetworkShortNames()` to get the string